.github/workflows/tests.yml

```diff
@@ -59,7 +59,9 @@ jobs:
       run: |
         source activate.sh
         export SNEK_DEBUG=1
-        pytest -v --runslow --cov-report=xml --cov-report=term-missing
+        pytest -v --cov-report=xml tests/test_make.py &
```

Comment:
It could be good to add a comment explaining why it is useful to run these tests like that.